src/db/migrations/20190325172457-create-users.js

```diff
@@ -29,7 +29,7 @@ export default {
       },
       password: {
         type: Sequelize.STRING,
-        allowNull: false,
+        allowNull: true,
```

Comment:
I see we are allowing the possibility of a user to sign in, without a password. Could you kindly tell why we are doing so?

Reply:
When a user registers with a third-party application like Facebook, his password would not be provided by Facebook. However, he is able to create/edit password on his profile.